src/de/uni_hamburg/informatik/swt/se2/mediathek/ui/ausleihe/AusleihWerkzeug.java

```diff
@@ -69,24 +75,29 @@ public class AusleihWerkzeug
      * @param medienbestand Der Medienbestand.
      * @param kundenstamm Der Kundenstamm.
      * @param verleihService Der Verleih-Service.
+     * @param vormerkService Der Vormerk-Service.
      * 
      * @require medienbestand != null
      * @require kundenstamm != null
      * @require verleihService != null
+     * @require vormerkService != null
      */
     public AusleihWerkzeug(MedienbestandService medienbestand,
-            KundenstammService kundenstamm, VerleihService verleihService)
+            KundenstammService kundenstamm, VerleihService verleihService,
+            VormerkService vormerkService)
     {
         assert medienbestand != null : "Vorbedingung verletzt: medienbestand != null";
         assert kundenstamm != null : "Vorbedingung verletzt: kundenstamm != null";
         assert verleihService != null : "Vorbedingung verletzt: verleihService != null";
+        assert vormerkService != null : "Vorbedingung verletzt: verleihService != null";
 
         _verleihService = verleihService;
+        _vormerkService = vormerkService;
 
         // Subwerkzeuge erstellen
         _kundenAuflisterWerkzeug = new KundenauflisterWerkzeug(kundenstamm);
         _medienAuflisterWerkzeug = new AusleiheMedienauflisterWerkzeug(
-                medienbestand, verleihService);
+                medienbestand, verleihService, vormerkService);
```

Comment:
Da AusleiheMedienauflisterWerkzeug zuvor Probleme gemacht hatte & ich es reverten musste zur vorherigen Version, bitte auch diese Änderung rückgangig machen, danach NUR diese Änderung zu 6-implementation-in-ausleihemedienauflisterwerkzeug hinzufügen. (Rebase bei 6-implementation-in-ausleihemedienauflisterwerkzeug, neuer commit mit dieser Änderung)

Reply:
Mit den Änderungen hätte alles funktioniert. Soll ich nur den aktuellen parameter entfernen?